### DISCUSSION TITLE
Time logic of data collection

### DISCUSSION THREAD

**Discussion by sada-68** [owid/covid-19-data#2683] [General]
Dear colleagues,

I hope all is well with you.

I wonder if you could please answer my following question.

"Does the data collection (in the public file) follow a certain time logic? I mean, what is the time interval between samples and their number?"

With regards

**Comment by lucasrodes** [comment]
Hi @sada-68,
sorry for the late reply.

You can find all the details at https://docs.owid.io/projects/covid/en/latest/data-pipeline.html
